plugins/@grouparoo/google-sheets/__tests__/integration/google-sheets-import.ts

```diff
@@ -408,10 +409,11 @@ describe("integration/runs/google-sheets", () => {
         expect(recordsCount).toBe(10);
 
         await run.updateTotals();
+
         expect(run.state).toBe("complete");
-        expect(run.importsCreated).toBe(10);
+        expect(run.importsCreated).toBe(0);
         expect(run.recordsCreated).toBe(0);
-        expect(run.recordsImported).toBe(10);
+        expect(run.recordsImported).toBe(0);
         expect(run.percentComplete).toBe(100);
       },
```

Comment:
this is a little strange - after running an import schedule, wouldn't you want to see imports created?

Reply:
I'm not really sure, but I think it's because it's updating. I get it from CSV tests.

Reply:
Ah! The test above (line 314) is the one that makes the imports the first time!